src/App.js

```diff
@@ -1,25 +1,64 @@
-import logo from './logo.svg';
+import React, {Component} from 'react'
 import './App.css';
+import moviesData from './data.js'
+import AllMovies from './AllMovies'
+import Movie from './Movie'
+import './home-btn-img.png'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      movies: [],
+      movieView: false,
```

Comment:
I'm sure this has changed with implementing router, but a few notes here: 
1. I love that you chose to handle this with state!
2. You might want to be a little broader with you naming in case there are more than two pages. `currentPage: 'home'`